src/jobs/push-to-app-catalog.yaml

```diff
@@ -9,10 +9,27 @@ parameters:
   chart:
     description: "Name of the chart inside helm directory to push to the App Catalog."
     type: "string"
+  chart_testing_config_file:
+    description: >-
+      Chart Testing config file
+      (see: https://github.com/helm/chart-testing#configuration)
+    type: string
+    default: ""
+  chart_testing_additional_args:
+    description: >-
+      Additonal command line arguments
+      (ignored if config_file is set)
+    type: string
+    default: ""
```

Comment:
A few points here:

- Are you really sure we need those parameters and can't have sane defaults everywhere instead? It's easy to expose new parameters but hiding them is a breaking change.
- If they are really needed: how about replacing this with path to actual files holding the configuration in the repo?
- Let's replace `_` with `-`. I know this is inconsistent across the repo but the later is what we should have everywhere.

Reply:
> * Are you really sure we need those parameters and can't have sane defaults everywhere instead? It's easy to expose new parameters but hiding them is a breaking change.
> * If they are really needed: how about replacing this with path to actual files holding the configuration in the repo?

On these points, I have removed `additional-args`, but kept the config file... The config file can/will exist in the repo of the chart and it's up to the chart maintainer to enable disable things as part of linting/testing. If config file is omitted, then it will run using defaults/

> * Let's replace `_` with `-`. I know this is inconsistent across the repo but the later is what we should have everywhere.

Done

Thanks

Reply:
How about having our custom config-we-want-to-use-everywhere defined here? Are there configurations that _have_ to be changed between repositories?

Reply:
Personally, I am not sure. Because it means that if an App maintainer needs to deviate from the set values, they will need to raise a PR here to get that changed. I don't think that gains us anything.

I would like to at least keep the option of using a config file.

Reply:
Deviation means lack of consistency which we value a lot. If something needs to be adjusted we should rather do that for all the apps (which is in the orb itself). Unless there is a good/known reason to configure it differently per app. Still if we can't anticipate it now and it turns out that we need per app adjustment we can expose a parameter later on.

Reply:
Cool... I have simplified further. The lint doesn't take any parameters and uses defaults (except checking for maintainers which don't really apply)